app/models/monster.rb

```diff
@@ -0,0 +1,6 @@
+class Monster < ApplicationRecord
+    validates :name, presence: true
+    validates :description, presence: true, length: { minimum: 20}
+    validates :image, presence: true
```

Comment:
Let's come back to images when file uploading has been added. We recommend first being able to create new Monsters through a form, then in a later feature add file uploading to the form